pkg/scheduler/extender/extender.go

```diff
@@ -436,23 +447,22 @@ func (e *Extender) createReservation(ctx context.Context, namespace string, name
 	return nil
 }
 
-func (e *Extender) prepareListOfRequestedNodes(nodes []coreV1.Node) []string {
-	requestedNodes := []string{}
-
+func (e *Extender) prepareListOfRequestedNodes(nodes []coreV1.Node) (nodeNames []string) {
+	nodeNames = []string{}
 	for _, node := range nodes {
 		n := node
-		nodeID, err := annotations.GetNodeID(&n, e.annotationKey, e.nodeSelector, e.featureChecker)
```

Comment:
Do you suggest to read NodeUUID in extender from annotations and in node-service from csibmnodes? From my side to have 2 interfaces is redundant and might leads to races in the future. Could we use the same interface here or create a task for it?

Reply:
Extender uses k8s nodes in order to get node UUID. GetNodeID and ObtainID uses different ways to get node UUIDs.